salt_stack/pillar/top.sls

```diff
@@ -4,23 +4,30 @@ base:
 
   'endpoint*':
     - endpoint
+    - chrony
 
   'cachemanager*':
     - cachemanager
+    - chrony
 
   'activities*':
     - activities
+    - chrony
 
   'ep-jenkins*':
     - endpoint-jenkins
+    - chrony

```

Comment:
Remove `chrony` from all jenkins entries. They all run in the APLCloud and probably already have a connection the APL time server.